edx_notifications/base_data.py

```diff
@@ -210,7 +210,7 @@ class DateTimeField(TypedField):
     Specialized subclass of TypedField(datetime) as a convienence
     """
 
-    _expected_types = [datetime]
+    _expected_types = [datetime, FakeDatetime]
```

Comment:
This seemed to be the smallest change to get around the issue. It rejected the object otherwise (in `__set__`, it raised a `TypeError` `TypeError: Field expected type of '[<type 'datetime.datetime'>]' got '<class 'freezegun.api.FakeDatetime'>'`). I guess no big harm in letting it stay.